includes/Core/Modules/Module.php

```diff
@@ -575,7 +575,7 @@ private function make_data_request( Data_Request $data ) {
 	 * @return array List with two elements, the first with the start date and the second with the end date, both as
 	 *               'Y-m-d'.
 	 */
-	protected function parse_date_range( $range, $multiplier = 1, $offset = 1, $previous = false, $weekday_align = false ) {
+	public function parse_date_range( $range, $multiplier = 1, $offset = 1, $previous = false, $weekday_align = false ) {
```

Comment:
The visibility of this method shouldn't change here, it can however be changed in `FakeModule` like `\Google\Site_Kit\Tests\Core\Modules\FakeModule::exception_to_error`

Reply:
Ah, ok got it!

Reply:
Fixed in [`476fd98` (#1822)](https://github.com/google/site-kit-wp/pull/1822/commits/476fd98d9ed7ad15f0d7d972a3a82d83bb875b98)